qemu/tests/cfg/check_block_size.cfg

```diff
@@ -88,3 +88,17 @@
             nic_mode = tap
             physical_block_size_stg = 512
             logical_block_size_stg = 512
+        - 4096_4096_cluster_install:
+            only s390x, ovmf
```

Comment:
when remove this line, pass on aarch64
So this scene is supported on arm and does not need to specify ovmf for arm.

` (1/1) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.check_block_size.4096_4096_cluster_install.extra_cdrom_ks.arm64-pci: PASS (1723.45 s)`

Reply:
Same as ppc